{"timestamp": 1555153247, "height": 46, "width": 190, "version": 2, "env": {"TERM": "xterm", "SHELL": "/bin/bash"}}
[0.573304, "o", "\u001b[0;34mNo config file found; using defaults\u001b[0m\r\n"]
[0.594248, "o", "\u001b[0;34m/root/hosts did not meet host_list requirements, check plugin documentation if this is unexpected\u001b[0m\r\n\u001b[0;34m/root/hosts did not meet script requirements, check plugin documentation if this is unexpected\u001b[0m\r\n"]
[0.594371, "o", "\u001b[0;34m/root/hosts did not meet yaml requirements, check plugin documentation if this is unexpected\u001b[0m\r\n\u001b[0;34m/root/hosts did not meet ini requirements, check plugin documentation if this is unexpected\u001b[0m\r\n\u001b[0;34m/root/hosts did not meet auto requirements, check plugin documentation if this is unexpected\u001b[0m\r\n"]
[0.594409, "o", "\u001b[1;35m [WARNING]: Unable to parse /root/hosts as an inventory source\u001b[0m\r\n\u001b[1;35m\u001b[0m\r\n"]
[0.594525, "o", "\u001b[1;35m [WARNING]: No inventory was parsed, only implicit localhost is available\u001b[0m\r\n\u001b[1;35m\u001b[0m\r\n"]
[0.597198, "o", "\u001b[1;35m [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'\u001b[0m\r\n\u001b[1;35m\u001b[0m\r\n"]
[0.939823, "o", "^C"]
[0.981708, "o", "\r\nPLAY [localhost] *****************************************************************************************************************************************************************************\r\n"]
[1.635923, "o", "^C"]
[1.636178, "o", "Process WorkerProcess-2:\r\n"]
[1.636645, "o", "\u001b[0;31m [ERROR]: User interrupted execution\u001b[0m\r\n\u001b[0;31m\u001b[0m\r\n"]
[1.642708, "o", "Traceback (most recent call last):\r\n"]
[1.642745, "o", "  File \"/usr/lib/python3.5/multiprocessing/process.py\", line 249, in _bootstrap\r\n    self.run()\r\n  File \"/usr/local/lib/python3.5/dist-packages/ansible/executor/process/worker.py\", line 118, in run\r\n    self._final_q\r\n  File \"/usr/local/lib/python3.5/dist-packages/ansible/executor/task_executor.py\", line 140, in run\r\n    res = self._execute()\r\n  File \"/usr/local/lib/python3.5/dist-packages/ansible/executor/task_executor.py\", line 612, in _execute\r\n    result = self._handler.run(task_vars=variables)\r\n  File \"/usr/local/lib/python3.5/dist-packages/ansible/plugins/action/command.py\", line 24, in run\r\n    results = merge_hash(results, self._execute_module(task_vars=task_vars, wrap_async=wrap_async))\r\n  File \"/usr/local/lib/python3.5/dist-packages/ansible/plugins/action/__init__.py\", line 843, in _execute_module\r\n    res = self._low_level_execute_command(cmd, sudoable=sudoable, in_data=in_data)\r\n"]
[1.642884, "o", "  File \"/usr/local/lib/python3.5/dist-packages/ansible/plugins/action/__init__.py\", line 962, in _low_level_execute_command\r\n    rc, stdout, stderr = self._connection.exec_command(cmd, in_data=in_data, sudoable=sudoable)\r\n"]
[1.643082, "o", "  File \"/usr/local/lib/python3.5/dist-packages/ansible/plugins/connection/local.py\", line 133, in exec_command\r\n    stdout, stderr = p.communicate(in_data)\r\n  File \"/usr/lib/python3.5/subprocess.py\", line 1072, in communicate\r\n    stdout, stderr = self._communicate(input, endtime, timeout)\r\n  File \"/usr/lib/python3.5/subprocess.py\", line 1715, in _communicate\r\n    ready = selector.select(timeout)\r\n  File \"/usr/lib/python3.5/selectors.py\", line 376, in select\r\n    fd_event_list = self._poll.poll(timeout)\r\nKeyboardInterrupt\r\n"]
